packages/sdk-react/src/components/receivables/InvoiceDetails/CreateReceivable/sections/components/ReminderSection/ReminderSection.test.tsx

```diff
@@ -0,0 +1,150 @@
+  const queryClient = new QueryClient();
+
+  const moniteMock = new MoniteSDK({
+    entityId: '123',
+    fetchToken: () =>
+      Promise.resolve({
+        access_token: '123213',
+        token_type: 'Bearer',
+        expires_in: 3600,
+      }),
+  });
+
+  const methods = useForm();
+  return (
+    <QueryClientProvider client={queryClient}>
+      <I18nProvider i18n={I18n}>
+        <MoniteProvider
+          locale={{
+            code: 'en-US',
+          }}
+          monite={moniteMock}
+        >
+          <MoniteScopedProviders>
+            <FormProvider {...methods}>{children}</FormProvider>
+          </MoniteScopedProviders>
+        </MoniteProvider>
+      </I18nProvider>
+    </QueryClientProvider>
+  );
+};
+
+describe('ReminderSection', () => {
+  describe('#FormValidation', () => {
+    test('should show error message when fields are empty and form is submitted', async () => {
+      console.log(
+        'Starting test: should show error message when fields are empty and form is submitted'
+      );
+
+      await act(async () => {
+        renderWithClient(
+          <Wrapper>
+            <ReminderSection disabled={false} />
+          </Wrapper>
+        );
+      });
+
+      console.log('Rendered ReminderSection component');
+
+      await waitFor(() => {
+        expect(screen.queryByText(/Loading.../)).not.toBeInTheDocument();
+      });
+    });
+  });
+
+  describe('ReminderDetails Component', () => {
+    const renderWithTheme = (
+      component: React.ReactElement,
+      mode: 'light' | 'dark' = 'light'
+    ) => {
+      const theme = createTheme({
+        palette: {
+          mode,
+        },
+      });
+
+      return render(<ThemeProvider theme={theme}>{component}</ThemeProvider>);
+    };
+
+    const mockDetails: ReminderDetail = {
+      id: 'overdue-reminder-1',
+      name: 'Payment 1',
+      created_at: '2023-07-21',
+      updated_at: '2023-07-21',
+      recipients: {
+        bcc: undefined,
+        cc: undefined,
+        to: undefined,
+      },
+      term_1_reminder: {
+        body: 'test body',
+        days_before: 1,
+        subject: 'test subject',
+      },
+      term_2_reminder: {
+        body: 'test body',
+        days_before: 1,
+        subject: 'test subject',
+      },
+    };
+
+    test('renders without crashing', () => {
+      renderWithTheme(<ReminderDetails details={mockDetails} />);
+      expect(screen.getByText('Payment 1')).toBeInTheDocument();
+      expect(screen.getByText('2023-07-21')).toBeInTheDocument();
+    });
+
+    test('does not render when details are empty', () => {
+      const { container } = renderWithTheme(
+        <ReminderDetails details={undefined} />
+      );
+      expect(container).toBeEmptyDOMElement();
+    });
+
+    test('renders correctly in light mode', () => {
+      renderWithTheme(<ReminderDetails details={mockDetails} />, 'light');
+      const iconColor = '#0000008F';
```

Comment:
It's already pretty irrelevant, in my opinion. Testing the color scheme is QA's job.

Reply:
Discussed directly to play with :  ` lighten(theme.palette.primary.main, 0.9)`,

Reply:
Fixed